webapp/src/containers/components/modals/configModal.tsx

```diff
@@ -0,0 +1,454 @@
+                                    placeholder='Enter the display name of your action'
+                                />
+                            </Form.Group>
+                            <Form.Group className='form-group'>
+                                <Form.Label>{'Channels Added to'}</Form.Label>
+                                <Form.Control
+                                    type='text'
+                                    placeholder='Enter the name of channels in which you want to add the new user'
+                                />
+                            </Form.Group>
+                            <Form.Group className='form-group'>
+                                <Form.Label>{'Action Successfull message'}</Form.Label>
+                                <Form.Control
+                                    type='text'
+                                    placeholder='Enter the success message on completion of action'
+                                />
+                            </Form.Group>
+                        </Form>
+                    </div>}
+                    {!props.config && !actionVisible && <div>
+                        <Button
+                            className='add-actions'
+                            onClick={handleActions}
+                        >{'Add actions'}</Button>
+                    </div>}
+
+                    {deleteVisible && <div className={deleteVisible ? 'fade-enter' : 'fade-exit'}>
+                        <p>{'Are you sure you would like to delete the action ?'}</p>
+                    </div>}
+                </Modal.Body>
+                <Modal.Footer>
+                    {configVisible &&
+                    <Button
+                        variant='secondary'
+                        onClick={handleClose}
+                    >
+                        {'Close'}
+                    </Button>}
+                    {actionVisible &&
+                    <Button
+                        variant='secondary'
+                        onClick={handleClose}
+                    >
+                        {'Cancel'}
+                    </Button>}
+                    {deleteVisible &&
+                    <Button
+                        variant='secondary'
+                        onClick={handleClose}
+                    >
+                        {'Cancel'}
+                    </Button>}
+                    {configVisible &&
+                    <Button
+                        variant='primary'
+                        onClick={handleClose}
+                    >
+                        {'Save changes'}
+                    </Button>}
+                    {actionVisible &&
+                    <Button
+                        variant='primary'
+                        onClick={handleClose}
+                    >
+                        {'Add action'}
+                    </Button>}
+                    {deleteVisible &&
```

Comment:
I think you can combine it with the conditional on line 420

Reply:
combining them in one conditional will force us to add extra css, also we have removed this button in next PR